client/post-editor/editor-revisions-list/header.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+function EditorRevisionsListHeader( { restoreRevision, translate } ) {
+	return (
+		<div className="editor-revisions-list__header">
+			<h3>{ translate( 'You\'re seeing the latest changes' ) }</h3>
+			<Button
+				className="editor-revisions-list__restore-revision"
+				onClick={ restoreRevision }
+				compact={ true }
+			>
+				{ translate( 'Restore this revision' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **74** times:
`translate( 'Restore This Revision' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).